File: Java/src/Faccat/Exercicio86.java
package Faccat;

import java.util.Scanner;

public class Exercicio86 {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        int[] A = new int[10];
        int i, j, k;

        for (i = 0; i < 10; i++) {
            System.out.println("Digite um número:");
            A[i] = scanner.nextInt();
        }

        for (i = 0; i < 9; i++) {
            for (j = i + 1; j < 10; j++) {
                if (A[i] > A[j]) {
                    k = A[i];
                    A[i] = A[j];
                    A[j] = k;
                }
            }
        }

        System.out.print("Vetor ordenado: ");
        for (i = 0; i < 10; i++) {
            System.out.print(A[i] + " ");
        }

        scanner.close();
    }
}

File: JavaScript/Faccat/Exercicio86.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const A = [];
let i, j, k;

function lerNumeros() {
  rl.question("Digite um número: ", function (input) {
    const valor = parseInt(input);

    A.push(valor);

    if (A.length < 10) {
      lerNumeros();
    } else {
      ordenarVetor();
    }
  });
}

function ordenarVetor() {
  for (i = 0; i < 9; i++) {
    for (j = i + 1; j < 10; j++) {
      if (A[i] > A[j]) {
        k = A[i];
        A[i] = A[j];
        A[j] = k;
      }
    }
  }

  process.stdout.write("Vetor ordenado: ");
  for (i = 0; i < 10; i++) {
    process.stdout.write(A[i] + " ");
  }

  rl.close();
}

lerNumeros();